heat/core/tensor.py

```diff
@@ -556,6 +556,23 @@ def __eq__(self, other):
         """
         return relations.eq(self, other)
 
+    def fabs(self, out=None):
+        """
+        Calculate the absolute value element-wise and return floating-point tensor.
```

Comment:
Please provide a short description explaining why we implement fabs, abs and absolute